chapters/competitive-apps.tex

```diff
@@ -0,0 +1,190 @@
+% !TeX root = ../main.tex
+\chapter{Konkurenční aplikace}
+
+Po prozkoumání podobného žánru her
+– kooperace, komunikace a rychlost reakcí –
+dostupných v obchodě Google Play,
+byly vybrány konkurenční herní aplikace.
+Tyto hry byly vybrány zejména podle popularity a počtu stažení v obchodě
+a podle subjektivně vnímaného potenciálu,
+který má daná hra k inspiraci pro tvorbu herní aplikace.
+
+V následujících sekcích budou popsány vybrané 4 konkurenční aplikace,
+včetně shrnutí kladů a záporů.
+
+\section{DUAL!}
+
+První vybranou hrou je \emph{DUAL!}.
+Tato hra využívá zejména prvky kooperace a rychlosti reakcí
+a je velmi akčně laděná.
+Jak říká první věta stránky hry \cite{seabaa_dual} v obchodě Google Play:
+\uv{Mezi lidmi, napříč obrazovkami.}
+
+Hra má několik módů.
+Jedním z módů je mód \uv{DUEL}, ve kterém dva hráči
+– každý na svém zařízení –
+bojují proti sobě.
+
+Herní plocha je rozdělena do dvou 2D částí.
+Každý hráč se může pohybovat pouze ve své části, na svém zařízení.
+Herní postava hráče se ovládá natáčením jeho displeje.
+Postava má zásobník střel a po klepnutí na plochu postava vystřelí.
+Při podržení se postupně ze zásobníku nabíjí střely
+a vystřelí se větší, ale trochu pomalejší, střela.
+Rychlým použitím gesta švihnutí se postava otočí.
+
+Pokud hráč vystřelí na své ploše,
+střela za chvíli doletí do protihráčovi části.
+Střele protihráče se dá buď takticky vyhnout,
+nebo zneškodnit vystřelením vlastní střely.
+
+Cílem hry je několikrát zasáhnout protihráče, a tím ho zneškodnit.
```

Comment:
Jednovětné odstavce (i jinde v textu) nevypadají moc dobře... Doporučuji buď rozepsat nebo sloučit.